src/engine.cpp

```diff
@@ -16,54 +16,69 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 #include <fstream>
 #include <functional>
 #include <random>
+#include <filesystem>
+
 //Create definitions of the RandomEngine class here
 //Methods:
 RandomEngine::RandomEngine():length((uint8_t)8),symb(false),mod(symb?73:63){
     //Reading 100 bytes of data from urandom file for seed generation.
     //this part is independent of user's input, hence defined in the default contructor.
     std::ifstream file("/dev/urandom", std::ios::in | std::ios::binary);
     if(!file){
-    std::cout << "Error opening file\n";
+    std::cout<<"Error opening file\n";
```

Comment:
This is a stray change. Do `git diff` before committing.

Reply:
Sure, sir, this won't happen again!